grizli/pipeline/auto_script.py

```diff
@@ -1892,6 +1904,18 @@ def preprocess(field_root='j142724+334246',  HOME_PATH='/Volumes/Pegasus/Grizli/
     # Clean up
     if clean:
         clean_prep(field_root=field_root)
+    # Rename instrument back to JWST stuff
+    # add an if statement later for this
+    if isJWST:
+        for file in direct['files']:
+            hdu = pyfits.open(file, mode='update')
+            hdu[0].header['INSTRUME'] = 'NIRISS'
```

Comment:
Save the original keyword above when you change it (e.g., `OINSTRUM`) and change it back to that, e.g., NIRCam / MIRI.

Reply:
Done here: 28b92ae72465cf463a9a89570daa152f57a0fe16